src/connectRoutes.js

```diff
@@ -115,7 +115,11 @@ export default (
   }
 
   const {
+    // Flow doesn't like defaulting to string or func here, although it does not
+    // complain if we _do not_ provide a default value
+    // $FlowIssue
```

Comment:
but im surprised that that they don't let u set a default value of one of the expected types. weird.